gwpy/table/gravityspy.py

```diff
@@ -151,8 +151,9 @@ def download(self, **kwargs):
                                  duration_values.repeat(
                                    len(images_for_download_ext), 0).flatten(
                                      )).T
-        images_for_download_ext = images_for_download_ext.as_matrix(
-                                       ).repeat(len(cols_for_download), 0)
+        images_for_download_ext = images_for_download_ext.values
+        images_for_download_ext = images_for_download_ext.repeat(len(cols_for_download),
+                                                                 0)
```

Comment:
Can this be combined as

```suggestion
        images_for_download_ext = images_for_download_ext.values.repeat(
            len(cols_for_download),
            0,
        )
```